src/pages/home/report/ReportActionItem.js

```diff
@@ -469,10 +469,21 @@ function ReportActionItem(props) {
         const parentReportAction = ReportActionsUtils.getParentReportAction(props.report);
         if (ReportActionsUtils.isTransactionThread(parentReportAction)) {
             return (
-                <MoneyRequestView
-                    report={props.report}
-                    shouldShowHorizontalRule={!props.shouldHideThreadDividerLine}
-                />
+                <ShowContextMenuContext.Provider
+                    value={{
+                        anchor: popoverAnchorRef,
+                        report: props.report,
+                        action: props.action,
+                        checkIfContextMenuActive: toggleContextMenuFromActiveReportAction,
+                    }}
+                >
+                    <OfflineWithFeedback pendingAction={props.action.pendingAction}>
```

Comment:
Coming from https://github.com/Expensify/App/issues/26235

This line brought back wrapping `MoneyRequestView` in `OfflineWithFeedback`, which was removed in 
https://github.com/Expensify/App/pull/25399. This might be a mistake in merge conflict resolution.

This caused a double application of the opacity reduction effect.